playwright/tests/auth.setup.ts

```diff
@@ -6,29 +6,64 @@ import path from "path";
 
 setup.describe.configure({ mode: "serial" });
 
+const sleep = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+
 const authenticateAndSaveState = async (
   request: APIRequestContext,
   email: string,
   password: string,
   filePath: string,
 ) => {
   const basicApiService = new BasicApiService(request);
+  let retries = 0;
+  const maxRetries = 3;
+  const delayBetweenRetries = 3000;
 
-  await basicApiService.logInUserViaApi({ email, password });
+  while (retries < maxRetries) {
```

Comment:
we can't login in a loop until API will pass - I don't think that fixes anything:

PW have nice guides how to do login stuff, in our case we need to login and share sessions across the workers: https://playwright.dev/docs/auth#moderate-one-account-per-parallel-worker

Reply:
We should login accordingly to the API limits, not fighting with them. When we reach limitations that means either doing something wrong or API has a bug